### ISSUE TITLE
Screen reader does not announce expanded/ collapsed state of left navigation controls in windows.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2615]
### Description

**User Impact:**
Users relying on screen readers or keyboard navigation are unable to understand the structure of the nested menu. They cannot intuitively or efficiently navigate between menu levels, leading to confusion and a breakdown in usability.

### Expected behavior

Screen reader should announce expanded/ collapsed state of left navigation controls in windows.

### Actual behavior

Screen reader does not announce expanded/ collapsed state of left navigation controls in windows.
Screen reader remains silent

**Refer Attachment:**
Screen reader does not announce expanded & collapsed state of left navigation controls in windows.png
Screen reader does not announce expanded & collapsed state of left navigation controls in windows.mp4

https://github.com/user-attachments/assets/7c843189-100b-46c5-9155-452a896b6efa
<img width="960" height="512" alt="Image" src="https://github.com/user-attachments/assets/7561b1b3-7f60-4a3e-877a-45824c27af1c" />

### Steps to reproduce

1. Open the URL: https://docsify-preview-git-fork-sy-records-fix-2599-docsifyjs.vercel.app/preview/#/?id=docsify in edge browser in windows.
2. Turn on narrator.
3. Press tab key to navigate to the Quick start present at the left navigation.
4. Verify that screen reader announces the expanded/ collapsed state of left navigation controls in windows or not.

### Environment

**Test Environment:**
OS: Windows 11 25H2 (26200.6584)
Browser: Edge (Version 142.0.3595.33)
Screen Reader: Narrator, NVDA (2025.2), JAWS (2025.2508.120)
URL: https://docsify-preview-git-fork-sy-records-fix-2599-docsifyjs.vercel.app/preview/#/?id=docsify

### Additional Information

- [x] Bug still occurs when all/other plugins are disabled?

**Comment by paulhibbitts** [comment]
Hi @msftedad , thanks for the follow-up. Right now the Maintainer team is focused on work related to the release of v5. Once that is done we hope to loop back and assess this issue.
